data-extraction/ingredients.json

```diff
@@ -0,0 +1,43 @@
+[
+
+{
+    "id": 1,
```

Comment:
correction: I misread, we should ADD a spoonacular-id to be consistent not change this one